components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -66,6 +66,7 @@
 import javax.xml.stream.XMLStreamException;
 
 import java.io.File;
+import java.lang.reflect.InvocationTargetException;
```

Comment:
Any reason for importing InvocationTargetException class here? I don't see any usages.